stonesoup/predictor/tests/test_kalman.py

```diff
@@ -3,11 +3,11 @@
 import pytest
 import numpy as np
 
-from ...models.transition.linear import ConstantVelocity
-from ...predictor.kalman import (
+from stonesoup.models.transition.linear import ConstantVelocity
```

Comment:
This change shouldn't be required.

Reply:
indeed it isn't. Wonder why I felt the need. Reverted.